[2025-03-04T04:00:32.972+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-04T04:00:32.994+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: solar_energy_crops_pipeline.extract_data scheduled__2025-03-03T00:00:00+00:00 [queued]>
[2025-03-04T04:00:33.003+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: solar_energy_crops_pipeline.extract_data scheduled__2025-03-03T00:00:00+00:00 [queued]>
[2025-03-04T04:00:33.003+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-04T04:00:33.021+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_data> on 2025-03-03 00:00:00+00:00
[2025-03-04T04:00:33.034+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=68) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-04T04:00:33.041+0000] {standard_task_runner.py:72} INFO - Started process 70 to run task
[2025-03-04T04:00:33.037+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'solar_energy_crops_pipeline', 'extract_data', 'scheduled__2025-03-03T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/mainDag.py', '--cfg-path', '/tmp/tmp9egd_ngk']
[2025-03-04T04:00:33.043+0000] {standard_task_runner.py:105} INFO - Job 24: Subtask extract_data
[2025-03-04T04:00:33.136+0000] {task_command.py:467} INFO - Running <TaskInstance: solar_energy_crops_pipeline.extract_data scheduled__2025-03-03T00:00:00+00:00 [running]> on host 57e0d7bfc1b4
[2025-03-04T04:00:33.305+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='solar_energy_crops_pipeline' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-03T00:00:00+00:00'
[2025-03-04T04:00:33.306+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-04T04:00:33.368+0000] {logging_mixin.py:190} INFO - Extracting data from API...
[2025-03-04T04:00:33.371+0000] {logging_mixin.py:190} INFO - Fetching data for year 1967
[2025-03-04T04:00:36.164+0000] {logging_mixin.py:190} INFO - Sleeping for a while to respect API rate limits...
[2025-03-04T04:01:06.169+0000] {logging_mixin.py:190} INFO - Fetching data for year 1968
[2025-03-04T04:01:08.762+0000] {logging_mixin.py:190} INFO - Sleeping for a while to respect API rate limits...
[2025-03-04T04:01:38.780+0000] {logging_mixin.py:190} INFO - Fetching data for year 1969
[2025-03-04T04:01:40.822+0000] {logging_mixin.py:190} INFO - Sleeping for a while to respect API rate limits...
[2025-03-04T04:02:10.827+0000] {logging_mixin.py:190} INFO - Fetching data for year 1970
[2025-03-04T04:02:12.554+0000] {logging_mixin.py:190} INFO - Sleeping for a while to respect API rate limits...
[2025-03-04T04:02:42.571+0000] {logging_mixin.py:190} INFO - Fetching data for year 1971
[2025-03-04T04:02:44.508+0000] {logging_mixin.py:190} INFO - Sleeping for a while to respect API rate limits...
[2025-03-04T04:03:14.524+0000] {logging_mixin.py:190} INFO - Fetching data for year 1972
[2025-03-04T04:03:16.566+0000] {logging_mixin.py:190} INFO - Sleeping for a while to respect API rate limits...
[2025-03-04T04:03:20.346+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-03-04T04:03:20.626+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-04T04:03:20.670+0000] {process_utils.py:132} INFO - Sending 15 to group 70. PIDs of all processes in the group: [70]
[2025-03-04T04:03:20.675+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 70
[2025-03-04T04:03:20.701+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-03-04T04:03:21.603+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-04T04:03:21.728+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=70, status='terminated', exitcode=0, started='04:00:32') (70) terminated with exit code 0
